pandas/tests/io/test_html.py

```diff
@@ -1233,3 +1234,10 @@ def run(self):
         while helper_thread1.is_alive() or helper_thread2.is_alive():
             pass
         assert None is helper_thread1.err is helper_thread2.err
+
+    def test_parse_Path_object(self):
```

Comment:
add a small doc string and reference the github issue. Something along the lines of:
```
"""
read_html should be able to cope with Path

GH 37705
"""
```

Reply:
We actually don't generally use docstrings in tests - you can replace the docstring with a simple `# GH 37705` comment

Reply:
ups, sorry. I got that wrong :)